### ISSUE TITLE
Dev test: Gravity Force Lab Basics 1.0.0-dev.39

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#333]
@terracoda, @emily-phet  @jessegreenberg @ariel-phet gravity-force-lab-basics 1.0.0-dev.39 is ready for dev testing. The goal is to publish this sim with full audio descriptions and sonification before the end of the quarter. Please document issues in https://github.com/phetsims/gravity-force-lab-basics/issues and link to this issue. This sim has never been published, but Gravity Force Lab has, and much of this code is shared. Most of the work done since original regular version publication has been outfitting with accessible features, and that will be the majority of testing for this issue also.

Assigning @ariel-phet for prioritization.

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, Chrome and Safari
- [x] Latest iOS, Safari
- [x] Windows 10, all browsers
- [x] Latest Chrome OS, Chrome

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-dev.39/phet/gravity-force-lab-basics_all_phet.html)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) Dev Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.

<h3>Focus and Special Instructions</h3>

- The number pickers are a new a11y component. Make sure that they sound good on all platforms/devices
- This sim should support mobile accessibility, please test that too
- Please make sure that audio descriptions sound the same when using a "refreshed" copy of the sim as they do when the reset all button was pressed. 

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-dev.39/phet/gravity-force-lab-basics_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-dev.39/phet/gravity-force-lab-basics_all_phet.html)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](link).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
Edge lags a bit when dragging the masses.  Doesn't seem to be a problem in the dev version of Edge.

**Comment by jessegreenberg** [comment]
@KatieWoe as part of this test could you also please test the sim with on an Android device with TalkBack? We are interested in supporting that assistive device too and would like to get a sense of how it behaves.

It should work much like VoiceOver in terms of gesture control.

**Comment by jessegreenberg** [comment]
Also, @KatieWoe can you please see if https://github.com/phetsims/ohms-law/issues/146 is an issue in this sim? We haven't been able to see the bug anywhere else yet, but now that we know it happens more consistently after waiting 2+ minutes I am wondering if we will see it here too.

**Comment by KatieWoe** [comment]
It does happen on RIAW master, but not on published.  It does happen on GFLB.  Good catch.

**Comment by KatieWoe** [comment]
@jessegreenberg TalkBack does not seem to work at all.  Unless I am doing something quite wrong.
https://drive.google.com/file/d/1H4cbq-0EKOu10eefASCAlrjX7RqLpHwt/view?usp=sharing

**Comment by KatieWoe** [comment]
QA is done

**Comment by zepumph** [comment]
>@jessegreenberg TalkBack does not seem to work at all. Unless I am doing something quite wrong.
https://drive.google.com/file/d/1H4cbq-0EKOu10eefASCAlrjX7RqLpHwt/view?usp=sharing

@KatieWoe talkback works very differently from traditional UX. Here is a tutorial, I did the first ~10 minutes or so and was able to use GFLB quite easily, let me know if you have any questions, and thanks for your patience.
https://support.google.com/accessibility/android/answer/6283677?hl=en

**Comment by jessegreenberg** [comment]
Thanks @zepumph, that is a great resource. One of the ways to read content with talkback (from the resource) is "Explore by touch" where you tap or drag your finger around the screen to hear what is there. This also works with mobile VO, so I expected it to work with TalkBack too, but it seems that it doesn't for our sims. The other ways to read content seem to be working on my Galaxy S6.

**Comment by jessegreenberg** [comment]
@KatieWoe can you comment on the status of this? I noticed that it was marked as done but it looks like there was still some testing being done on this more recently, is this still complete?

**Comment by KatieWoe** [comment]
It is done.  Sometimes when there are no other issues in the repo, we go back and revisit recently finished issues, which is why there are new issues.  Alternatively, we might find something while confirming a fixed issue.  One of these was an issue I was discussing on slack during the test, and decided to move to github after.

**Comment by jessegreenberg** [comment]
Great, thanks for the update. @zepumph can this issue be closed the?

**Comment by zepumph** [comment]
Thanks, this is ready for closing.
